model_analyzer/plots/detailed_plot.py

```diff
@@ -72,12 +72,13 @@ def __init__(self, name, title, bar_width=0.5):
             )
         ]
 
+        # Okabe-Ito guideline colors for colorblind
```

Comment:
Add reference link as comment to this?

Reply:
Hi! The URI that I used was: https://jfly.uni-koeln.de/color/ (I found the link at https://siegal.bio.nyu.edu/color-palette/)

Reply:
Added